lib/Controller/RestrictionController.php

```diff
@@ -83,22 +83,25 @@ public function update(
 		int $maxRooms,
 		array $roomTypes,
 		int $maxParticipants,
-		bool $allowRecording
+		bool $allowRecording,
+		bool $allowLogoutURL
```

Comment:
Probably a c+p error. Should be `$logoutURL`.

Reply:
Can you explain to me why? This function updates the values of the restrictions, which is stored in $allowLogoutURL.

Reply:
Sorry, my fault.